change/@microsoft-teams-js-27d23c6b-df82-4548-81df-6efc35e67daa.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Proxy save and remove events to child windows",
```

Comment:
>Proxy save and remove events to child windows [](http://example.com/codeflow?start=14&length=45)

For consistency in the public changefile, please use past tense. Something like this:

Enabled proxying of save and remove events in the `pages.config` capability to child windows